ci/install-extra-builddeps.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/bash
+# Install build dependencies not in the cosa buildroot already
+set -xeuo pipefail
+if ! command -v cxxbridge; then
+    cargo install --root=/usr cxxbridge-cmd
```

Comment:
Does this actually need to be in `/usr` BTW? Running this script locally.

Reply:
Hmm, I guess we could dispatch on `$(getuid)`?